### ISSUE TITLE
Image Card: Repetitive Links

### ISSUE THREAD

**Issue by sirrah-tam** [ithaka/pharos#509]
**The problem**
There are repeated links within the image card that to the same destination, one for the image and one for the title. This creates additional tab stops (# of items in keyboard focus) that over time will lengthen the amount of time required by a keyboard-only or screen reader user to interact with the site.

**The solution**
Ideally, there should be a single link for an "item" that takes users to the detail page. If possible, have the items title become the keyboard focusable link, and set up the click event for the image as a separate mouse-only action that does not receive focus. 

**Alternatives considered**
There are also ways to expand the 'clickable' area of a link through CSS. 

**Additional information**
Additionally, depending on the implementation, the Pharos documentation should include information about not have the item's container gain keyboard focus unless there is some action that can be taken while receiving focus, ie. pressing Enter to go to detail page, using arrow keys to move to the next item, etc. If this approach is taken, then neither the image or title would gain focus after the outer container, and would move directly to any other inner links (author/collection).

**Comment by daneah** [comment]
@sirrah-tam is this a duplicate of #353? If there's nuance between them, might we reconcile them together?

**Comment by sirrah-tam** [comment]
Aha, yes that is exactly it! Should this one be closed and include my comments on the other one?

**Comment by daneah** [comment]
@sirrah-tam That sounds good!

**Comment by sirrah-tam** [comment]
Closed as duplicate of #353
